addons/calendar/migrations/11.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# © 2017 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def update_user_ids(cr):
+    """
+    Now user_id is required, set it to the one who created the entry
+    """
+    cr.execute("""
+        UPDATE calendar_contacts SET user_id = create_uid
+        WHERE user_id IS NULL;
+    """)
+
+
+def cleanup_translations(cr):
```

Comment:
I insist this should be an openupgradelib because it's normal to use this same code across several migration scripts.

Reply:
Not sure what you mean by 'insist' but I think it's a good suggestion!

Reply:
hehe, it was for @yuriqp because he has done it in another PR and I have put the same comment.

Reply:
As I mentioned in other PR (https://github.com/OCA/OpenUpgrade/pull/1188) @rruebner already created a PR with this function in openupgradelib (OCA/openupgradelib#87). What should I do, wait until it is merged and then create a new PR to use it in this migration?

Reply:
When merged, we make another PR calling that function instead.